ingest.py

```diff
@@ -21,9 +21,52 @@
 RECORD_MANAGER_DB_URL = os.environ["RECORD_MANAGER_DB_URL"]
 
 
+def _get_text(tag):
+    new_line_elements = {"h1", "h2", "h3", "h4", "code", "p", "li"}
```

Comment:
* Maybe markdown to retain relative information about headers? (e.g., h1, h2, h3, h4 to `#` , `##`, ..)
* Should we expect newline after `<div>`?
* not sure about the node hiearchy, not relevant here, but in general good to check if html comments / script / css picks up  txt as well